Gemfile

```diff
@@ -1,11 +1,21 @@
 source 'https://rubygems.org'
 
-ruby '2.5.0'
+ruby '2.6.0'
```

Comment:
Your Ruby version is 2.5.0, but your Gemfile specified 2.6.0

Reply:
![screenshot 2019-01-26 at 08 44 05](https://user-images.githubusercontent.com/40179292/51784813-9a336e80-2146-11e9-9b32-a951ddab2f83.png)
As in the screenshot, I double checked and can confirm that I updated  the ruby version on my laptop before doing the Chitter Challenge, so at the time of the challenge it was already 2.6.0.